### ISSUE TITLE
StartPortForwardingSession: parameter to keep connection active

### ISSUE THREAD

**Issue by rdisipio** [aws/amazon-ssm-agent#253]
The StartPortForwardingSession command is awesome. However, I noticed that the tunnel gets closed if the connection drops, which in my case is an intended behaviour (e.g. I query a postgres db multiple times during the execution of a python script). I understand this feature was added in version 1.1.33.0 of the Session Manager plugin (see [here](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#plugin-version-history)).

Is there a solution (not a workaround) to prevent the tunnel to get closed, _e.g._ by adding a parameter?

```
  aws ssm start-session --target ${TARGET} --document-name AWS-StartPortForwardingSession --parameters '{"portNumber":["5432"],"localPortNumber":["5433"],"dropConnection":["False"]}'
```

**Comment by sahirangaraju** [comment]
Thanks for reaching out. With the latest version of SSM Agent and Session Manager Plugin, tunnel disconnect issue should be resolved. Can we please get the agent and plugin versions you are using?

**Comment by rdisipio** [comment]
I am using version:

```
$ aws --version
 aws-cli/1.16.260 Python/3.7.5 Darwin/18.7.0 botocore/1.12.250
```

**Comment by AmyZhangAmz** [comment]
Hi  @rdisipio:

Thank you for your interest about  Amazon-ssm-agent. We've released the 2.3.714.0  to Ubuntu Snap as requested. Feel free to let us know if you have any question.

**Comment by jupeter** [comment]
Have same issue. On client:
